### PR TITLE
Russian translation updated

### PR DESCRIPTION
**Pull request by arvicco**

### CODE REVIEW

File: _translations/ru.yml [hunk 1]
```diff
@@ -294,7 +302,7 @@ ru:
...
     legal: "Легальность"
     islegal: "Легален ли Биткойн?"
-    islegaltxt1: "Насколько нам известно, <a href=\"https://en.wikipedia.org/wiki/Legality_of_bitcoin_by_country\">Биткойн не является незаконным</a> с точки зрения действующего законодательства в большинстве юрисдикций. Тем не менее, в некоторых юрисдикциях (например в Аргентине и России) законодательно введены строгие ограничения или запреты на использование иностранных валют. Другие юрисдикции (например Таиланд) могут требовать лицензирования определенных субъектов, таких как биткойн-обменники."
+    islegaltxt1: "Насколько нам известно, <a href=\"http://bitlegal.io/\">Биткойн не является незаконным</a> с точки зрения действующего законодательства в большинстве юрисдикций. Тем не менее, в некоторых юрисдикциях (например в Аргентине и России) законодательно введены строгие ограничения или запреты на использование иностранных валют. Другие юрисдикции (например Таиланд) могут требовать лицензирования определенных субъектов, таких как биткойн-обменники."
```
Comment (harding) [inline _translations/ru.yml]: @arvicco This should link to the Wikipedia page, as we changed it in pull #765

**Comment by arvicco** [comment]
For some reason, this link is stuck in Transifex original English text. I do not change original links upon translations, and have no power to change original English text as uploaded there.

**Comment by arvicco** [comment]
I mean, I can change it manually in my downstream but it won't solve the problem as the next translator will be stuck with this link all over again.

**Comment by saivann** [comment]
@arvicco @harding This probably means this English string hasn't been updated upstream yet, but will be updated eventually (this one should actually be updated in all translations by writing a quick script and usng the Transifex client, by either me or @harding). Meanwhile it's expected that @arvicco wouldn't have noticed the change, and there is no harm to fix the link in the Russian translations only as of now.

**Comment by arvicco** [comment]
@harding @saivann Updated manually as requested.

**Comment by harding** [comment]
@arvicco hey, I just wanted to let you know that I'm slowly trying to figure out our translation system, which is why I haven't merged this yet.  I spent half an hour on it today, and I'll keep spending half an hour on it each day until I feel confident I'm not going to break something. :-)

Thanks for taking the time to make these translations!

**Comment by arvicco** [comment]
@harding No big deal, there are no major parts of Russian-language site broken at the moment, translations mostly cover new wallets and confidentiality features.

**Comment by harding** [comment]
@arvicco merged.  Thanks for all of your work and for your patience with me!  (Also some extra thanks for translating the legal policy; I suspect that must've been rather boring.)

**Pull request merged**